src/config-parser.js

```diff
@@ -351,6 +351,30 @@ function createSourceMapDirectory(sourceMapPath) {
   d(`Using separate sourcemap path at ${sourceMapPath}`);
 }
 
+function versionToFloat(ver) {
+  return parseFloat(ver.replace(/^([^\.]\.[^\.])\..*$/, '$1'));
+}
```

Comment:
One would hope that if Node changed the behavior of `parseFloat`, there'd be a **lot** of people angry, not just us.